.github/workflows/test_e2e_ibf_system.yml

```diff
@@ -0,0 +1,97 @@
+# See: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions
+
+name: 'Test: E2E (IBF-System)'
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - '.github/workflows/test_e2e_ibf_system.yml'
+      - 'e2e/**'
+      - './.env.example'
+      - '!**.md'
+      - './'
+
+env:
+  e2eTestsPath: './e2e'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v4
+
+      - name: Set up Node.js version
+        uses: actions/setup-node@v4
+        with:
+          node-version-file: '${{ env.e2eTestsPath }}/.node-version'
+          cache: 'npm'
+          cache-dependency-path: '${{ env.e2eTestsPath }}/package-lock.json'
+
+      - name: Install E2E-Tests code-dependencies
+        working-directory: ${{ env.e2eTestsPath }}
+        run: |
+          npm ci --omit=optional --no-fund --no-audit
+
+      - name: Lint E2E-Tests code
+        working-directory: ${{ env.e2eTestsPath }}
+        run: 'npm run lint'
+
+      - name: Set ENV-variables for test-environment
+        run: |
+          cp example.env .env
+
+      - name: Install API-service code-dependencies
```

Comment:
@gulfaraz good question. It seemed necessary, because the e2e tests are run from the host with 'npm test', and they in turn use some of the API-service files (e.g. utility.helper.ts) to reset the DB, which in turn needs the 'supertest' package. And without this 'npm install' step on the host instead of within the container, this package was not found.

Do you think this setup should be changed?

Reply:
It shouldn't be that the installation is needed on the host machine while the service is running inside docker.

I will investigate this before I propose a change.